tests/cypress/integration/mysite.home.spec.ts

```diff
@@ -57,4 +62,118 @@ describe('Enable sitemap on MySite', () => {
+            .clickOnSave()
+            .validateSucessMessage()
+            .clickBack()
+        // Step #2 publishing the site and flush the cache
+        siteHomePage.publishSite('My Site').clickPublishAll().flushCache()
+
+        // Need to enable SEO on "Search Result"
+        siteHomePage
+            .editPage('Search Results')
+            .clickOnSEOSitemap()
+            .clickOnDedicatedSitemap()
+            .clickOnSave()
+            .validateSucessMessage()
+            .clickBack()
+        siteHomePage.publishSite('My Site').clickPublishAll().flushCache() // publish the whole site and flush
+
+        // Getting the response from sitemap.index.xml
+        // according to the sitemap.xml spec,
+        // the url value should reside in a <loc /> node
+        // https://www.google.com/sitemaps/protocol.html
+        cy.requestFindNodeInnerHTMLByName('sites/mySite/sitemap.index.xml', 'loc').then((urls) => {
+            expect(urls.length).to.be.equal(2)
+            urls.forEach((url) => {
+                const regexArray = getUrlInfoWithHost(String(url)) // Typescript issue that need to explicitly convert to String
+                if (regexArray != null && regexArray.length > 4) {
+                    const path = regexArray[5]
+                    expect(this.fixture.siteMapPath).to.include(path)
+                }
+
+                cy.requestFindXMLElementByTagName(String(url), 'url').then((urlGroups) => {
+                    expect(urlGroups.length).to.be.equal(1) // Only one sitemap url entry in each
+                    Cypress.$(urlGroups).each(($idx, $list) => {
+                        const pageUrl = $list.getElementsByTagName('loc')
+                        const lastMod = $list.getElementsByTagName('lastmod')
+                        expect(pageUrl.length).to.be.equal(1)
+                        const siteUrlArray = getUrlInfoWithHost(pageUrl[0].innerHTML)
+                        expect(siteUrlArray.length).to.be.greaterThan(4)
+                        expect(lastMod.length).to.be.equal(1)
+                    })
+                })
+            })
+        })
+    })
+
+    it('Exclude pages from sitemap', function () {
+        // Testcase C3218089 "Exclude pages from sitemap"
+        const siteUrlHome = '/jahia/page-composer/default/en/sites/mySite/home.html'
+        // Loading the fixtures
+        cy.fixture('C3218089/sitemap.index.urlpath.json').then((json) => {
+            this.fixture = json // Recommended from Cypress as no real good way to get promise variable out (use this variable)
+        })
+
+        // Step #1 the standard enable
+        siteHomePage
+            .goTo(siteUrlHome)
+            .editPage('My Site')
+            .clickOnSitemap()
+            .clickOnSave()
+            .validateSucessMessage()
+            .clickBack()
+        // Step #2 publishing the site and flush the cache
+        siteHomePage.publishSite('My Site').clickPublishAll().flushCache()
```

Comment:
I think we are missing a step here, in Testrail we have :
_- Open page composer and edit page /home/our-companies
- Open Live Roles and remove Guest & users from readers
- Add bill as user
- Save_

Reply:
I simplified the portion here
As I don't see any reason we need to add the user and save
By going through the SEO portion is enough for testing the no-index